homeassistant/components/homewizard/config_flow.py

```diff
@@ -110,6 +111,32 @@ async def async_step_zeroconf(
 
         return await self.async_step_discovery_confirm()
 
+    async def async_step_dhcp(
+        self, discovery_info: DhcpServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle dhcp discovery to update existing entries.
+
+        This flow is triggered only by DHCP discovery of known devices.
+        """
+        try:
+            device = await self._async_try_connect(discovery_info.ip)
+        except RecoverableError as ex:
+            _LOGGER.error(ex)
```

Comment:
Should we even log this on error level?
As in, it might even scare the user at this point in time :)

Reply:
Well, it is a situation that _should_ never happen. When it does (if even possible) I think it is nice to have something traceable.

But I am absolutely okay to remove this!